var/spack/repos/builtin/packages/libspatialite/package.py

```diff
@@ -44,6 +44,7 @@ class Libspatialite(AutotoolsPackage):
     depends_on("iconv")
     depends_on("librttopo", when="@5.0.1:")
     depends_on("libxml2")
+    depends_on("libxml2@:2.12", when="@:5.1.0")
```

Comment:
Is this true in newer versions too? Is there an upstream bug report or PR that fixed this?

Reply:
I tried building latest libspatialte, 5.1.0 with latest libxml2, 2.13.5. Still getting the same build issue. May be I can clarify the reason for this constraint in the PR.

Reply:
found this. may be improving libxml2 package will be sufficient.
https://www.gaia-gis.it/fossil/libspatialite/tktview/2b1697e601

Reply:
Let's try adding an `+http` variant and changing the dep to `libxml2+http`

Reply:
it worked. I'll update the PRs.

Reply:
now the issue is fixed properly.